sklearn/metrics/_scorer.py

```diff
@@ -942,6 +959,9 @@ def check_scoring(estimator, scoring=None, *, allow_none=False):
                 "to a scorer." % scoring
             )
         return get_scorer(scoring)
+    if isinstance(scoring, (list, tuple, set, dict)):
+        scorers = _check_multimetric_scoring(estimator, scoring=scoring)
+        return _MultimetricScorer(scorers=scorers)
```

Comment:
One thing that is occluded is the `raise_exc: bool = True` argument of `_MultimetricScorer`. I have personally never used it so I do not mind. Is the decision not to support passing it in?

Reply:
I did not want to increase the scope of this PR. Usually increasing the scope makes it harder to merge.

If we want to add `raise_exc`, then it can be done in a follow up PR.